app/views/options.html

```diff
@@ -31,9 +34,45 @@
     </label>
     <p>Example: {{10 * options.velocityUnit().mult|longnum}} meat/{{options.velocityUnit().name}}</p>
   </div>
+  <div class="form-group">
```

Comment:
Commit message says the login's still a work in progress; that's fine. I assume you're still working on a dropbox login that doesn't need the user to copy app/oauth keys around.

If you'd like, I'll pull your changes now if you add an `ng-if="env.isDebugEnabled()"` around this, to prevent it from appearing in production until everything's done.